feature_engine/timeseries/forecasting/expanding_window_features.py

```diff
@@ -117,6 +117,28 @@ class ExpandingWindowFeatures(BaseForecastTransformer):
     pandas.expanding
     pandas.aggregate
     pandas.shift
+
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> from feature_engine.timeseries.forecasting import ExpandingWindowFeatures
+    >>> X = pd.DataFrame(dict(date = ["2022-09-18",
+    >>>                             "2022-10-27",
```

Comment:
picky I know: could we make the indentation nicer? I think the "2022- etc should be aligned with the one on the first line.

Reply:
hahaha. My editor thinks that is the correct indentation, that is why I left it that way. But sure, I can fix it. No worries.